cookbooks/aws-parallelcluster-environment/resources/network_service/network_service_alinux2023.rb

```diff
@@ -17,7 +17,25 @@
 end
 
 use 'partial/_network_service'
+use 'partial/_network_service_redhat_based'
 
 def network_service_name
-  'systemd-resolved'
+  'systemd-networkd'
+end
+
+action :restart do
+  log "Restarting 'systemd-networkd systemd-resolved' service, platform #{node['platform']} '#{node['platform_version']}'"
+
+  execute "Reload system configuration files before restarting services" do
+    command "systemctl daemon-reload"
+  end
+
+  %w(systemd-networkd systemd-resolved).each do |service_name|
```

Comment:
Why do we need to restart `systemd-resolved` if the network service is `systemd-networkd`?

Reply:
Nvm answered by the comment below